test/vignette/http/integration_test.clj

```diff
@@ -16,97 +19,97 @@
 (def system-local (create-system lis))
 
 (with-state-changes [(before :facts (start system-local default-port))
-                     (after :facts (stop system-local))]
+                     (after :facts (do (stop system-local)))]
```

Comment:
The `do` was actually a mistake. I didn't mean to commit that. Typically `do` is used when you want to group multiple expressions that have side effects. Something like:

```
(do
  (execute-something)
  (stop system-local))
```